packages/features/tasker/tasks/sendEmail.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from "zod";
+
+const sendEmailPayloadSchema = z.object({
+  /**  */
+  to: z.string(),
+  /** The email template to send */
+  template: z.string(),
+  payload: z.string(),
+});
+
+export async function sendEmail(payload: string): Promise<void> {
+  try {
+    const parsedPayload = sendEmailPayloadSchema.parse(JSON.parse(payload));
+    console.log(parsedPayload);
```

Comment:
Do we want to add this to our logs?

Reply:
RN there's no created sendEmail tasks yet. We're starting with only webhooks for now.